docs/CONTRIBUTING.rst

```diff
@@ -0,0 +1,186 @@
+.. _contributing:
```

Comment:
We no longer add this to the formulas themselves, since it's available in the `.github` repo.

Reply:
remove?

Reply:
Yes, please.